doc/conf.py

```diff
@@ -272,4 +272,4 @@
         "https://documen.tician.de/pytools": None,
         }
 
-autoclass_content = "both"
+autoclass_content = "class"
```

Comment:
Note to self: I feel like this might have broken something, so need to check.

Reply:
I agree that `class` is the desired setting, FWIW.

Reply:
Looked through it and fixed some issues in cc5873b. Should be ok now.